dev/com.ibm.ws.jpa_spec10_injection_fat/publish/servers/JPAServerDMI/hibernate-cfg.xml

```diff
@@ -0,0 +1,24 @@
+<server>
```

Comment:
instead of duplicating these config fragments for each server, you could put them at ` dev/com.ibm.ws.jpa_spec10_injection_fat/publish/shared/config/*.xml` (which gets automatically copied over by the FAT framework) and then include them from the primary config using the `${shared.config.dir}` variable as documented here: https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/rwlp_dirs.html

Reply:
Great idea. Implemented.